praktikumsplaner-frontend/frontend/src/stores/error.ts

```diff
@@ -0,0 +1,30 @@
+import { defineStore } from "pinia";
+
+export interface ErrorState {
+    title: string | undefined;
+    message: string | undefined;
+    show: boolean;
```

Comment:
maybe visible is better. `show` is more an action than a property. `show`, `hide` or `setVisible`  are possible actions.